src/main/java/com/example/onlinebookstore/model/Order.java

```diff
@@ -0,0 +1,59 @@
+package com.example.onlinebookstore.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+import java.util.HashSet;
+import java.util.Set;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.JdbcTypeCode;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.Where;
+import org.hibernate.type.SqlTypes;
+
+@Entity
+@Data
+@SQLDelete(sql = "UPDATE orders SET is_deleted = true WHERE id=?")
+@Where(clause = "is_deleted=false")
```

Comment:
```suggestion
@SQLRestriction("is_deleted=false")
```
@Where is is deprecated